runtime/catalog-karapace/src/main/java/io/aklivity/zilla/runtime/catalog/karapace/internal/KarapaceEventContext.java

```diff
@@ -39,40 +40,43 @@ public class KarapaceEventContext
     private final EventFW.Builder eventRW = new EventFW.Builder();
     private final KarapaceEventExFW.Builder karapaceEventExRW = new KarapaceEventExFW.Builder();
     private final int karapaceTypeId;
-    private final int remoteAccessRejectedEventId;
-    private final int futureCompletedExceptionallyId;
-    private final int staleSchemaServedId;
+    private final int unretrievableSchemaSubjectVersionId;
+    private final int staleSchemaID;
+    private final int unretrievableSchemaId;
+    private final int retrievableSchemaSubjectVersionId;
+    private final int retrievableSchemaId;
     private final MessageConsumer eventWriter;
     private final Clock clock;
 
     public KarapaceEventContext(
         EngineContext context)
     {
         this.karapaceTypeId = context.supplyTypeId(KarapaceCatalog.NAME);
-        this.remoteAccessRejectedEventId = context.supplyEventId("catalog.karapace.remote.access.rejected");
-        this.futureCompletedExceptionallyId = context.supplyEventId("catalog.karapace.future.completed.exceptionally");
-        this.staleSchemaServedId = context.supplyEventId("catalog.karapace.stale.schema.served");
+        this.unretrievableSchemaSubjectVersionId = context.supplyEventId("catalog.karapace.unretrievable.schema.subject.version");
+        this.staleSchemaID = context.supplyEventId("catalog.karapace.unretrievable.schema.subject.version.stale.schema");
+        this.unretrievableSchemaId = context.supplyEventId("catalog.karapace.unretrievable.schema.id");
+        this.retrievableSchemaSubjectVersionId = context.supplyEventId("catalog.karapace.retrievable.schema.subject.version");
+        this.retrievableSchemaId = context.supplyEventId("catalog.karapace.retrievable.schema.id");
         this.eventWriter = context.supplyEventWriter();
         this.clock = context.clock();
     }
 
-    public void remoteAccessRejected(
+    public void unretrievableSchemaSubjectVersion(
```

Comment:
I think it might be more readable if these methods were named `onXxx` instead of `xxx`.

So in the calling context it would be something like `events.onUnretrievableSchemaSubjectVersion(...)` (not `event...`), wdyt?